plugins/plugin-debugger/che-plugin-debugger-ide/src/main/java/org/eclipse/che/plugin/debugger/ide/debug/DebuggerPresenter.java

```diff
@@ -243,7 +255,7 @@ protected void updateStackFrameDump(long threadId) {
     }
   }
 
-  protected void updateVariables(long threadId, int frameIndex) {
+  protected void updateVariablesAndExpressions(long threadId, int frameIndex) {
```

Comment:
Split into two different methods.
The first one will update variables, the second one will update watch expressions